brew.sh

```diff
@@ -3,15 +3,30 @@
 # Install Brew if not installed
 command -v brew > /dev/null || /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-# Update Brew and Formulae
+# Update Brew and Formula
 brew update
 
 # Install brew bundle
 brew tap Homebrew/bundle
 
-# Install tools and apps from the Brewfile wih Brew and Cask
-brew bundle --file=Brewfile
-pecl install xdebug
+# Install tools and apps for Devolepers or Non-Developers
+echo "Type the job role ( Developer or Non-Developer ), followed by [ENTER]:"
+read role
+
+while [[ "$role" != "Developer" && "$role" != "Non-Developer" ]]
+do
+  echo "Please type the correct format"
+  read role
+done
+
+if [ "$role" != "Non-Developer" ]
+then
+  brew bundle --file=Developer
+  pecl install xdebug
+else
+  brew bundle --file=Non-Developer
```

Comment:
We could probably add an extension to these files as otherwise they would usually be considered readmes or makefiles. E.g. `Developer.bundle` and `Non-Developer.bundle`.

Reply:
@hkdobrev Good call! I will make that change.